jwk/jwt_strategy.go

```diff
@@ -113,7 +114,7 @@ func (j *RS256JWTStrategy) refresh(ctx context.Context) error {
 		return err
 	}
 
-	public, err := FindPublicKey(keys)
+	public, err := FindKeyByPrefix(keys, "public")
```

Comment:
I reverted this because we still rely on the private: pattern to be available for private keys.